locales/nl.json

```diff
@@ -12,7 +12,7 @@
 	"would turn %s off": "schakelt %s uit",
 	"would turn %s on": "shakelt %s in",
 	"if": "als",
-	"Edit rule": "Regels bewerken",
+	"Edit rule": "Regel bewerken",
```

Comment:
I think I might be wrong, this text isn't for the menu but somewhere else so your translation is correct. My bad.